test/unit/webdriver/search_context/windows_test.py

```diff
@@ -13,7 +13,9 @@
 # limitations under the License.
 
 import httpretty
+from selenium.webdriver.common import by
```

Comment:
I assume this particular was never executed and is invalid. The `-windows uiautomation` locator type is not supported on the server side and was removed from other clients.

Reply:
Yes. this lib also had a deprecation mark.